src/go/k8s/internal/controller/redpanda/redpanda_controller.go

```diff
@@ -215,21 +206,23 @@ func (r *RedpandaReconciler) Reconcile(c context.Context, req ctrl.Request) (ctr
 		}
 	}
 
-	if rp.Spec.Migration != nil && rp.Spec.Migration.Enabled {
-		if err := r.tryMigration(ctx, log, rp); err != nil {
-			log.Error(err, "migration")
-		}
-	}
+	// TODO There needs to be tests to validate migration from operator v1 to operator v2 (Cluster to Redpanda and maybe vice versa)
```

Comment:
I feel like we're well within our rights to drop support for this migration by now. WDYT?

Reply:
Would that be problematic for cloud consumption of v1 APIs and potentially migrating to v2?

Reply:
Yes, we can drop it as the only known user of this feature was cloud team. They don't need to move atm, so ✂️